lib/greet.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+module.exports = exports = {};
+var name = process.argv[2];
+
+exports.greet = function(name) {
+  if (arguments.length === 0) throw new Error ('name not included');
+  console.log(`hello ${name}`);
+  return (`hello ${name}`);
+};
+exports.favPlace = function(place) {
+  if (arguments.length === 0) throw new Error ('place not included');
+  return (`your favorite place is ${place}`);
+};
+
+exports.greet(name);
```

Comment:
Line 16 isn't necessary in this case, you can just invoke the function in index.js and pass the argument it's expecting :) I noticed in index.js you are invoking greet.greet(process.argv[2]) you'll want to keep things consistent and pass it the argument (name). Since you already set name equal to process.argv[2] in greet.js, you should be able to just pass it (name) when you invoke it.